app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -48,7 +49,7 @@
 
     <%= render "govuk_publishing_components/components/layout_header", {
       environment: "public",
-      search: true,
+      search: !without_search,
```

Comment:
I'm wondering if this could be rewritten to avoid the `!`. Would it be worth turning this on its head so the parameter is `hide_search` and then in the template itself the logic is `unless hide_search`?

Reply:
Not sure I've implemented this exactly the way you suggested, but I've refactored a bit. 
So now we have `search: show_search`, where `show_search` defaults to `true` (and can be set to false via local assigns).

Reply:
Sounds good, thanks.